kitsune/users/management/commands/remove_expired_registration_profiles.py

```diff
@@ -1,12 +0,0 @@
-from django.core.management.base import BaseCommand
```

Comment:
scripts/cron.py#job_remove_expired_registration_profiles

But do we want to run this once after we deploy this?